src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerIT.java

```diff
@@ -30,6 +31,7 @@
     webEnvironment = SpringBootTest.WebEnvironment.MOCK,
     classes = SandboxApplication.class)
 @AutoConfigureMockMvc
+@Disabled
```

Comment:
this test I am particular concern, that they are disabled, probably we can handle them in another ticket now it should be easy that the testcontainers exist so the context should be in a certain way easy to load and also to include the queues here.

Reply:
There is a ticket for this. 
https://europeana.atlassian.net/browse/MET-4553 
I'm happy to work on it when the time comes.